spec/rubocop/cop/naming/rescued_exceptions_variable_name_spec.rb

```diff
@@ -113,6 +113,100 @@
           RUBY
         end
       end
+
+      context 'with lower letters class name' do
+        it 'does not register an offense' do
+          expect_no_offenses(<<~RUBY)
+            begin
+              something
+            rescue my_exception
+              # do something
+            end
+          RUBY
+        end
+      end
+
+      context 'with method as `Exception`' do
+        it 'does not register an offense without variable name' do
+          expect_no_offenses(<<-RUBY.strip_indent)
```

Comment:
Could you replace `<<-RUBY.strip_indent` with `<<~RUBY` in this PR?

Reply:
I've replaced `strip_indent` with `<<~`. Thanks!